README.md

```diff
@@ -1,104 +1,102 @@
 # Vulhub - Docker-Compose files for vulnerabilities environment
 
+[中文版本(Chinese version)](README.zh-cn.md)
+
 [![Docker Stars](https://img.shields.io/badge/docker%20stars-20k-blue.svg)](https://hub.docker.com/u/vulhub/) [![GitHub](https://img.shields.io/github/license/vulhub/vulhub.svg)](https://github.com/vulhub/vulhub/blob/master/LICENSE)
  [![Chat on Discord](https://img.shields.io/discord/485505185167179778.svg)](https://discord.gg/GhMB3Z) [![Backers and sponors on Patreon](https://img.shields.io/badge/sponsor-patreon-73d6a1.svg)](https://www.patreon.com/phith0n) [![Backers and sponors on Opencollective](https://img.shields.io/badge/backer-opencollective-f89a76.svg)](https://www.patreon.com/phith0n)
 
-Vulhub是一个面向大众的开源漏洞靶场，无需docker知识，简单执行两条命令即可编译、运行一个完整的漏洞靶场镜像。
+Vulhub is an open source vulnerability lab for public. You don't need any docker knowledge, just execute two simple commend that you can compile and run a complete vulnerability environment mirror.
```

Comment:
No docker knowledge is required, just execute two simple commands and you have a vulnerable environment.